packages/flutter/test/cupertino/text_form_field_row_test.dart

```diff
@@ -339,7 +369,7 @@ void main() {
           child: CupertinoTextFormFieldRow(
             onChanged: (String value) {
               _called = true;
-              expect(value, state.value);
+              changeValue = value;
```

Comment:
I am a little worried that this will be a breaking change.

Reply:
```dart
  void didChange(T? value) {
    setState(() {
      _value = value;
      _hasInteractedByUser.value = true;
    });
    Form.of(context)?._fieldDidChange();
  }
```
the state value and the `Form.onChange` update together.

Reply:
```dart
void onChangedHandler(String value) {
              field.didChange(value);
              if (onChanged != null) {
                onChanged(value);
              }
            }
```
shouldn't the didChange called first before this onchange is called?

Reply:
This patch aims to call this onchange before didchange.

Reply:
ah so you change this line, too?

I thought this pr is aiming at the Form.onchange is called after the field value changes, not the CupertinoTextFormFieldRow.onChange? or am I missing something?

Reply:
I really want to adjust it after onchange . Do you think this is valuable?

Reply:
If you move onChanged to the front,  the formFieldState.value will not be updated before the onChanged is called. I am more toward to keep it as is

Reply:
You're right, I will double check is there a way that set value first and then call onchange, and call form onchange at last.